pkg/sqlcache/informer/listoption_indexer.go

```diff
@@ -63,7 +74,23 @@ const (
 	matchFmt                 = `%%%s%%`
 	strictMatchFmt           = `%s`
 	escapeBackslashDirective = ` ESCAPE '\'` // The leading space is crucial for unit tests only '
-	createFieldsTableFmt     = `CREATE TABLE "%s_fields" (
+
+	createEventsTableFmt = `CREATE TABLE "%s_events" (
```

Comment:
For people like me who jump right into the sql statements, a quick comment that "rv" stands for "resource version" would be helpful. Don't need much more than that.  Even something like

```
// rv: resourceVersion
… CREATE TABLE "%s_events" (
                       rv TEXT NOT NULL,
…
```
would be fine. The codebase sometimes uses `rv` to indicate `return value`, sometimes `reflect.ValueOf(...)`, so just the one comment will help readers like me quickly disambiguate

Reply:
Good idea, I added a comment. Resolving, feel free to re-open.